### DISCUSSION TITLE
useAPIKey: true breaks user auth

### DISCUSSION THREAD

**Discussion by maxibenner** [payloadcms/payload#8800] [Q&A] [answered]
I am trying to add the API key option on the user collection but whenever I change `auth: true` to `auth: {useAPIKey: true}`, I am no longer able to access the dashboard. I believe I am following the documentation here but would be happy for any pointers.

**Comment by akhrarovsaid** [comment, marked as answer]
Hey @maxibenner,

I've just tested in the monorepo `auth` folder, and can confirm that the `auth: { useAPIKey: true, ... }` option does not seem to impact the dashboard in the way you're describing. Should work as expected. I'd love to help you, however you've provided such a small amount of information that it's hard for me to know where the root of your problem comes from. 

Can you, first and foremost, create a working reproduction of your issue? I'm sure the devs would appreciate that the most. If you can't seem to reproduce the issue, then it's likely a misconfiguration. Your Paylaod version would also be helpful as well as any error messages.

**Comment by maxibenner** [comment]
Hey @akhrarovsaid,

Thank you for the quick response! While trying to create a working reproduction I realized that my issue is likely connected to me not properly syncing changes with migrations. I got it working for now. I won't get deeper into it here since this was most definitely a fault on my end. Thank you for your help!
